tests/manage/mcg/test_multi_region.py

```diff
@@ -27,12 +27,14 @@ class TestMultiRegion:
 
     @tier1
     @pytest.mark.polarion_id("OCS-1599")
-    def test_multiregion_bucket_creation(self, mcg_obj, multiregion_mirror_setup):
+    def test_multiregion_bucket_creation(
+        self, mcg_obj, multiregion_mirror_setup_session
+    ):
         """
         Test bucket creation using the S3 SDK
         """
 
-        mirrored_bucket_name = multiregion_mirror_setup[0].name
+        mirrored_bucket_name = multiregion_mirror_setup_session[0].name
```

Comment:
I am not sure if this won't collide with https://github.com/red-hat-storage/ocs-ci/blob/master/tests/ecosystem/upgrade/test_noobaa.py#L40.

Reply:
Great catch, thank you :)